merlin/util/duplicatebcremoval.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+from scipy.spatial import cKDTree
+import networkx as nx
+import pandas as pd
+from random import choice
+
+
+def cleanup_across_z(barcodes: pd.DataFrame, zPlanes: int, maxDist: float) \
+        -> pd.DataFrame:
+    """ Depending on the separation between z planes, spots from a single
+        molecule may be observed in more than one z plane. These putative
+        duplicates are removed based on supplied distance and z plane
```

Comment:
It could be useful to have a unit test for this function to make sure it behaves correctly in the edge cases and that we don't break it in the future.